app/services/search/google_search_service.rb

```diff
@@ -1,21 +1,103 @@
 # frozen_string_literal: true
 
+require 'user_agent_randomizer'
+
 module Search
   class GoogleSearchService
     GOOGLE_SEARCH_URL = 'https://www.google.com/search?q=%{keyword}&hl=%{language}&lr=%{language}'
 
+    USER_AGENT_HEADER = 'User-Agent'
+    USER_AGENT_TYPE = 'desktop_browser'
+
+    KEY_DIV_TOP_ADWORDS_CONTAINER = 'div.pla-unit-container'
```

Comment:
In terms of naming, you might prefer the work `SELECTOR` as in DOM selector. Also it might be better to create an object containers all elements:

```ruby
SELECTORS = {
  adwords_top: 'div.pla-unit-container'
	adwords_body: 'div.v5yQqb'
}
```

It is similar to how selectors are structured in UI tests (see https://github.com/nimblehq/mia-web/blob/development/spec/systems/mia_campaign/reject_campaign/reject_campaign_spec.rb#L4)